plugins/devtools/README.md

```diff
@@ -0,0 +1,375 @@
+# DevTools
+
+Welcome to the DevTools plugin!
```

Comment:
```suggestion
Welcome to the DevTools plugin!  This plugin helps elevate useful information about a Backstage instance, which can be helpful for an integrator for troubleshooting, review, and understanding their installation.
```

oh maybe on more tweak; just to help identify who the TARGET audience for this is? assuming it's not for "the mass of users", but likely just those few on an integrator team who are responsible for the care/feeding ?

Reply:
I went with "reviewing" but if you feel differently let me know